packages/@react-aria/virtualizer/src/Virtualizer.tsx

```diff
@@ -94,13 +100,15 @@ function Virtualizer<T extends object, V>(props: VirtualizerProps<T, V>, ref: Re
     }
   }, [onLoadMore, state]);
 
+  let lastContentSize = useRef(0);
   useLayoutEffect(() => {
     if (!isLoadingRef.current && onLoadMore && !state.isAnimating) {
-      if (state.contentSize.height > 0 && state.contentSize.height <= state.virtualizer.visibleRect.height) {
+      if (state.contentSize.height !== lastContentSize.current && state.contentSize.height > 0 && state.contentSize.height <= state.virtualizer.visibleRect.height) {
```

Comment:
This extra check will prevent the Virtualizer from calling loadMore multiple times until there is enough content to fill the Virtualizer's visible rect. This means if the first loadMore doesn't load enough content to make the virtualizer scrollable, then the user can't trigger any further loadMore calls (see https://reactspectrum.blob.core.windows.net/reactspectrum/1413f7c3690cb18a6655328eaf20221de79ca7f2/storybook/index.html?path=/story/tableview--async-loading-server-filtering-load-more&providerSwitcher-express=false)

By the time isLoadingRef becomes false, the `state.contentSize.height` has already settled and `lastContentSize` has already been set equal to it. There seems to be one time where the `state.contentSize.height` doesn't match `lastContentSize` but `isAnimating` is true then, meaning the `onLoadMore` doesn't trigger. Do we need `state.contentSize.height !== lastContentSize.current`?

Reply:
Without that check, two tests fail:

* ComboBox › async › onLoadMore is called on initial open
* ComboBox › async › onLoadMore is not called on when previously opened

In both cases, `onLoadMore` is called twice instead of once. Looks like the first one is a change from main, where it was only called once before. Not sure what is correct.